nb/api/skeleton.md

```diff
@@ -0,0 +1,21 @@
+[index](../../nb/api/index.md)
+### skeleton()
+Parameter|Default|Type
+---|---|---
+
+Generates a set of skeletal segments from the volumes of the input geometry.
```

Comment:
What would these be used for?

Reply:
There are a couple of possibilities.

* Finding excessively thin parts of a model and impose a minimum diameter.
* Producing armatures for articulation.
* Segmentation of models into main parts.

Not sure how well it will work in practice yet, though.

Reply:
Very cool ideas :)